### PR TITLE
NEW Input to close existing issues

### PR DESCRIPTION
**Pull request by emteknetnz**
Issue https://github.com/silverstripe/.github/issues/243

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,3 +19,17 @@ steps:
...
+
+## Inputs:
+
+### Title
+The title of the issue to create. Required.
+`title: Title of my issue`
+
+### Description
+The description of the issue to create. Required.
+`composer_install: true`
```
Comment (ChloeSartorelli) [inline README.md]: I'm guessing that was a copy/paste mistake? 😅 Should give an example of `description`, not `composer_install`

File: action.yml [hunk 2]
```diff
@@ -8,11 +8,49 @@ inputs:
...
+        RESP_CODE=$(curl -w %{http_code} -s -o __issues.json \
+        -X GET https://api.github.com/repos/$GITHUB_REPOSITORY/issues?state=open \
+        -H "Accept: application/vnd.github+json" \
+        -H "Authorization: token ${{ github.token }}" \
+        )
+        if [[ $RESP_CODE != "200" ]]; then
+          echo "Unable to fetch issues - HTTP response code was $RESP_CODE"
+          exit 1
+        fi
+        ISSUE_NUMBERS=$(jq -r --arg TITLE "$TITLE" '.[] | select(.title == $TITLE) | .number' __issues.json)
```
Comment (ChloeSartorelli) [inline action.yml]: If feasible, we should check the user that created the issue as well. We only want to close auto-created issues.

**Review by GuySartorelli — CHANGES_REQUESTED** [review]

File: action.yml (hunk 2 above)
Comment (emteknetnz) [reply action.yml]: Done

File: README.md (hunk 1 above)
Comment (emteknetnz) [reply README.md]: Done

File: README.md [hunk 3]
```diff
@@ -19,3 +19,17 @@ steps:
...
+
+## Inputs:
+
+### Title
+The title of the issue to create. Required.
+`title: Title of my issue`
+
+### Description
+The description of the issue to create. Required.
+`description: true`
```
Comment (ChloeSartorelli) [inline README.md]:
```suggestion
`description: Main body of the issue goes here`
```
It's not boolean

**Review by GuySartorelli — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (emteknetnz) [reply README.md]: lol

Comment (emteknetnz) [reply README.md]: fixed

**Review by GuySartorelli — APPROVED** [review]
LGTM

**Comment by ChloeSartorelli** [comment]
Tagged as `1.1.0`

**Pull request merged**